src/olympia/devhub/templates/devhub/versions/file_status_message.html

```diff
@@ -1,4 +1,4 @@
-<strong>{{ _('File {file_id} ({platform})')|format_html(file_id=file_id, platform=platform) }}</strong>
+<strong>{{ _('File {file_id} ({platform})')|format_html(file_id=fileid, platform=platform) }}</strong>
```

Comment:
Drive-by fix eh?

Reply:
😉 
Its probably been broken since it was written X years ago, and only uncovered after #5837